cli/command/environment.go

```diff
@@ -139,13 +125,21 @@ func (e *EnvironmentCommand) create(c *cli.Context) error {
 	req := models.CreateEnvironmentRequest{
 		EnvironmentName:  args["ENVIRONMENT_NAME"],
 		InstanceType:     c.String("type"),
-		MinScale:         c.Int("min-scale"),
-		MaxScale:         c.Int("max-scale"),
+		Scale:            c.Int("scale"),
+		EnvironmentType:  config.DefaultEnvironmentType,
 		UserDataTemplate: userData,
 		OperatingSystem:  c.String("os"),
 		AMIID:            c.String("ami"),
 	}
 
+	if c.IsSet("scale") {
+		req.EnvironmentType = models.EnvironmentTypeStatic
+	}
+
+	if !c.IsSet("os") {
```

Comment:
We don't need to do this check since we already have the default specified when we create the flag.

Reply:
I thought so too but when running the test, the default wasn't being picked up for some reason. I'll double check what the issue could be.

Reply:
ah, that's an issue with `testutils.NewTestContext`. There's some functionality that `urfave/cli` does to allow defaults that I haven't found out how to duplicate yet. Would be awesome if you could figure it out :wink:

Reply:
I'll create a new issue for this. We're not testing the command correctly and not passing the args correctly either. I have a fix for this but we might as well update all the cli tests with the slightly tweaked pattern.
✅